src/wristpy/core/config.py

```diff
@@ -9,3 +11,22 @@ class Settings(pydantic_settings.BaseSettings):
     LIGHT_THRESHOLD: float = 0.03
     MODERATE_THRESHOLD: float = 0.1
     VIGOROUS_THRESHOLD: float = 0.3
+
+    LOGGING_LEVEL: int = 20
```

Comment:
Given @ReinderVosDeWael comment on slack, should this be set to 10 for DEBUG?

Reply:
In my conception of it, during the  normal course of running wristpy, the debug logs we write should be in the code, but not actually logging, so as not to flood the user with unnecessary information. When something goes wrong and we need to get in there and debug, you would switch the Settings().LOGGING_LEVEL to 10 yourself, and all those debug logs you wrote will now work. That was my reasoning for having 20 (INFO) as the default.